lib/Runtime/Library/JsBuiltIn/JsBuiltIn.js

```diff
@@ -1,4 +1,4 @@
-//-------------------------------------------------------------------------------------------------------
+﻿//-------------------------------------------------------------------------------------------------------
```

Comment:
Looks like your editor added a BOM here, which got copied to the bytecode files as well and triggered the ASCII check.

Reply:
dc65f464a0085adeffeb2395fb68b391bb147ce3